docs/data/data-grid/custom-columns/EditingWithDatePickers.tsx

```diff
@@ -44,24 +40,12 @@ const dateColumnType: GridColTypeDef<Date, string> = {
   })),
   valueFormatter: (value) => {
     if (value) {
-      return dateAdapter.format(value, 'keyboardDate');
+      return format(value, 'MM/dd/yyyy', { locale });
```

Comment:
Prefer date library usage over using Pickers package internals as an example for end users. 🙈

Reply:
+1